proto/aserto/directory/model/v3/model.proto

```diff
@@ -7,43 +7,50 @@ option go_package = "github.com/aserto-dev/go-directory/aserto/directory/model/v
 import "google/api/field_behavior.proto";
 import "google/protobuf/empty.proto";
 
-import "aserto/directory/common/v3/common.proto";
-
 service Model {
-    rpc GetManifest(GetManifestRequest) returns (stream GetManifestResponse) {};
-    rpc SetManifest(stream SetManifestRequest) returns (SetManifestResponse) {};
+    rpc GetModel(GetModelRequest) returns (stream GetModelResponse) {};
```

Comment:
The artifact this API interacts with is a manifest file, not the model, which is the hydrated expanded result of loading the manifest file. This name is intentional. Please use GetManifest and SetManifest.